content/src/controller/Controller.ts

```diff
@@ -396,34 +396,28 @@ export class Controller {
       includeOverwrittenInfo: includeAuthChain
     }
 
-    const {
-      pointerChanges: deltas,
-      filters,
-      pagination
-    } = await this.components.sequentialExecutor.run('GetPointerChangesEndpoint', () =>
-      getPointerChanges(this.components, {
-        filters: requestFilters,
-        offset,
-        limit,
-        lastId,
-        sortBy
-      })
+    const { pointerChanges, filters, pagination } = await this.components.sequentialExecutor.run(
+      'GetPointerChangesEndpoint',
+      () =>
+        getPointerChanges(this.components, {
+          filters: requestFilters,
+          offset,
+          limit,
+          lastId,
+          sortBy
+        })
     )
-    const controllerPointerChanges: ControllerPointerChanges[] = deltas.map((delta) => ({
-      ...delta,
-      changes: Array.from(delta.changes.entries()).map(([pointer, { before, after }]) => ({ pointer, before, after }))
-    }))
 
-    if (controllerPointerChanges.length > 0 && pagination.moreData) {
-      const lastPointerChange = controllerPointerChanges[controllerPointerChanges.length - 1]
+    if (pointerChanges.length > 0 && pagination.moreData) {
+      const lastPointerChange = pointerChanges[pointerChanges.length - 1]
       pagination.next = this.calculateNextRelativePathForPointer(lastPointerChange, pagination.limit, filters)
     }
 
-    res.send({ deltas: controllerPointerChanges, filters, pagination })
+    res.send({ deltas: pointerChanges, filters, pagination })
   }
 
   private calculateNextRelativePathForPointer(
-    lastPointerChange: ControllerPointerChanges,
+    lastPointerChange: DeploymentPointerChanges,
```

Comment:
`DeploymentWithAuthChain` is the same type as  `DeploymentPointerChanges` can we keep only `DeploymentWithAuthChain` for maximum compatibility with snapshot-fetcher and to prevent accidental divergences?

Reply:
Sure, done!